--- ./libcine/daemon.c.org	Tue May 15 19:49:40 2001
+++ ./libcine/daemon.c	Wed Oct 16 09:19:57 2002
@@ -10,19 +10,21 @@
 void do_daemon(void)
 {
 #ifndef WIN32
-  switch (fork()) {
-    case -1: 
-      exit (-1);
-    case 0:
-      break;
-    default:
-      exit(0);
-  }
-  /* child continues */
-  setsid();    /* release controlling terminal */
+	int i;
+	switch (fork()) {
+		case -1: 
+			exit (-1);
+		case 0:
+			break;
+		default:
+			exit(0);
+	}
+	/* child continues */
+	for(i=0;i<255;i++) close(i);
+	setsid();    /* release controlling terminal */
 #else
-  debug("do_daemon", "Minimizing the window\n");
-  //ShowWindow(GetForegroundWindow(), SW_HIDE);
+	debug("do_daemon", "Minimizing the window\n");
+	//ShowWindow(GetForegroundWindow(), SW_HIDE);
 #endif
-  umask(0);    /* clear out file mode creation mask */
+	umask(0);    /* clear out file mode creation mask */
 } /* do_daemon */
